en/developer_guide/install.rst

```diff
@@ -1,149 +1,111 @@
+Install the database
+~~~~~~~~~~~~~~~~~~~~
 
-To install composer with curl:
+Install mongo in the 2.6 version at least :
 
 .. code-block:: bash
 
-    $ curl -sS https://getcomposer.org/installer | php
+    $ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 7F0CEB10
+    $ echo 'deb http://downloads-distro.mongodb.org/repo/debian-sysvinit dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list
+    $ sudo aptitude update
+    $ sudo aptitude install mongodb-org
 
-If you don't have curl installed, you can also download it with php:
+Check the mongo version :
 
 .. code-block:: bash
 
-    $ php -r "readfile('https://getcomposer.org/installer');" | php
+    $ mongo --version #MongoDB shell version: 2.6.*
 
-see [Download Composer](https://getcomposer.org/download/)
+Install NPM
+~~~~~~~~~~~
 
-Install OpenOrchestra
----------------------
-
-Install open-orchestra with composer:
+As a prerequisite, install those packages :
 
 .. code-block:: bash
 
-    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
-    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
+    $ sudo aptitude install wget curl make g++ gcc libcurl4-openssl-dev libsasl2-2 libsasl2-dev libcurl3
 
-Clone the provisioning repository :
+Download and install npm :
 
 .. code-block:: bash
 
-    $ git clone git@github.com:open-orchestra/open-orchestra-provision.git
-
-Install roles from ansible-galaxy
----------------------------------
+    $ wget http://nodejs.org/dist/v0.10.24/node-v0.10.24.tar.gz ./node-v0.10.24.tar.gz
+    $ tar -xvzf node-v0.10.24.tar.gz
+    $ cd node-v0.10.24
+    $ sudo ./configure --prefix=/usr/local/ && sudo make && sudo make install
 
-Install roles needed to launch the box
-as a prerequisite, update your python modules if required with those two
+Install the project
+-------------------
 
-.. code-block:: bash
-
-    $ aptitude install python-yaml
-    $ aptitude install python-jinja2
-
-If running under Mac OS X, you would install them through `pip`
+Download Composer
+~~~~~~~~~~~~~~~~~
 
-    easy_install pip
-    pip install pyyaml jinja2
+Composer is the package manager used by modern PHP applications.
 
-Then go into openorchestra directory
+To install composer with curl:
 
 .. code-block:: bash
 
-    $ ansible-galaxy install --role-file=provisioning/galaxy.yml
-
-Override the dns redirection
-----------------------------
+    $ sudo aptitude install git
+    $ curl -sS https://getcomposer.org/installer | php
 
-In the `/etc/hosts` file of your computer add the following lines :
+If you don't have curl installed, you can also download it with php:
 
-.. code-block:: text
+.. code-block:: bash
 
-    192.168.33.10   admin.openorchestra.dev
-    192.168.33.10   front.openorchestra.dev
-    192.168.33.10   demo.openorchestra.dev
-    192.168.33.10   media.openorchestra.dev
+    $ php -r "readfile('https://getcomposer.org/installer');" | php
```

Comment:
In that doc, there's a link between where you run that command and the directory where you install Orchestra (as there's no copy of composer.phar) Maybe the doc could say where to run that command in preview of the orchestra install